boot/boot.go

```diff
@@ -169,6 +170,23 @@ type NameAndRevision struct {
 	Revision snap.Revision
 }
 
+// DebugDumpBootvars writes a dump of the bootvars to the given writer
+func DebugDumpBootvars(w io.Writer) error {
+	bloader, err := bootloader.Find("", nil)
+	if err != nil {
+		return err
+	}
+	allKeys := []string{"snap_mode", "snap_core", "snap_try_core", "snap_kernel", "snap_try_kernel"}
```

Comment:
lk also has `reboot_reason` and `bootimg_file_name` though both seem to be an internal implementation detail.

Is `snap_menuentry` worth displaying here too?